driver_cpl/bld/namelist_files/namelist_defaults_drv.xml

```diff
@@ -4,6 +4,275 @@
 
 <namelist_defaults>
 
+<!-- =========================================  -->
```

Comment:
You mean deleted on the ESMCI side?  Yes that's true.  As @mvertens said elsewhere, we just need to add the one thing we did change to the new file.